.changeset/forty-onions-deliver.md

```diff
@@ -0,0 +1,6 @@
+---
+'@redocly/openapi-core': patch
+'@redocly/cli': patch
+---
+
+Added ignoreCase option for tags-alphabetical rule
```

Comment:
Oh, actually I think this should be a minor change, as this adds a new functionality. @justnero what do you think?

```suggestion
---
'@redocly/openapi-core': minor
'@redocly/cli': minor
---

Added `ignoreCase` option for `tags-alphabetical` rule.
```

Reply:
Agreed, wasn't quite sure about this one.